tests/python/unittest/test_contrib_operator.py

```diff
@@ -444,6 +445,46 @@ def test_modulated_deformable_convolution():
                         else:
                             rtol, atol = 0.05, 1e-3
 
+@with_seed()
+def test_dynamic_reshape():
+    def dynamic_reshape_testcases(src_shape, shape_arg, dst_shape):
+        data = mx.sym.Variable('data')
+        shape = mx.sym.Variable('shape')
+        net = mx.sym.contrib.dynamic_reshape(data, shape)
+        js = net.tojson()
+        net = mx.sym.load_json(js)
+        dat_npy = np.random.rand(*src_shape)
+        grad_npy = np.random.rand(*dst_shape)
+        args = {
+            'data': mx.nd.array(dat_npy),
+            'shape': mx.nd.array(shape_arg)
+            }
+        args_grad = {
+            'data': mx.nd.empty(src_shape)
+        }
+        exe = net.bind(default_context(), args, args_grad)
+        exe.forward(is_train=True)
```

Comment:
Does this operator only work with symbol API?

Reply:
works with ndarray too.

Reply:
Can you add an ndarrray test then?